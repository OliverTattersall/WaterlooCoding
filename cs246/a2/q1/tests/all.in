r a 11101/
r b 10110/
r c 010101/
r d 10101/
pa,
pb,
pc,
pd,
da
db
dc
dd
+ a 10/
+ b 01/
pa,
pb,
pc,
pd,
da
db
dc
dd
q
