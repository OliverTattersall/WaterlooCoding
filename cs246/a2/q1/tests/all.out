1,1,1,0,1

1,0,1,1,0

0,1,0,1,0,1

1,0,1,0,1

29
22
21
21
1,1,1,0,1,1,0

1,0,1,1,0,0,1

0,1,0,1,0,1

1,0,1,0,1

118
89
21
21
